cluster-autoscaler/cloudprovider/konvoy/konvoy_manager.go

```diff
@@ -97,61 +118,31 @@ func (k *KonvoyManager) GetNodeGroupSize(nodeGroup string) (int, error) {
 	return len(nodes.Items), nil
 }
 
-// GetNodeGroupTargetSize returns the size of the node group as a sum of current
-// observed size and number of upcoming nodes.
-func (k *KonvoyManager) GetNodeGroupTargetSize(nodeGroup string) (int, error) {
-	k.nodeGroupQueueSizeLock.Lock()
-	defer k.nodeGroupQueueSizeLock.Unlock()
-	realSize, err := k.GetNodeGroupSize(nodeGroup)
-	if err != nil {
-		return realSize, err
+// GetNodeGroupTargetSize returns the target size of the node group.
+func (k *KonvoyManager) GetNodeGroupTargetSize(nodeGroupName string) (int, error) {
+	konvoyCluster := &kommanderv1beta1.KonvoyCluster{}
+	konvoyCluster.Name = k.clusterName
+	clusterNamespacedName := types.NamespacedName{
+		Namespace: "kommander",
```

Comment:
This has to be `konvoy`

Reply:
Let's add a flag to allow passing the namespace (that will work in the future when we add autosdcaling for kommander konvoy clusters with diff namespace names)